docs/schema/guidance/repr-state-owned-enterprises.rst

```diff
@@ -0,0 +1,99 @@
+.. _representing-soes:
+
+Representing state-owned enterprises
+========================================
+
+.. highlights::
+
+    **Key requirement:** If an entity's status as a state-owned enterprise needs to be represented, then its Entity statement MUST be the subject of Ownership-or-control statement(s) connecting it, either directly or indirectly, to an Entity statement with ``entityType`` of 'state' or 'stateBody'.
+
+
+Overview
+------------------------
+
+The definition of 'state-owned enterprise' (SOE) can change according to jurisdiction or domain. The Beneficial Ownership Data Standard (BODS) does not therefore rely on, or support, any one definition. Instead, BODS enables the interests between a state (or state body) and an enterprise to be represented. This requires the state itself (or state body) to be represented by an Entity statement.
+
+In particular, the following properties and values SHOULD be used to represent information about enterprises and their links to states:
+
+* ``entityType`` SHOULD be set to 'state' or 'stateBody', and ``jurisdiction`` SHOULD be used to represent the particular state.
+* ``entitySubtype`` SHOULD be used to refine the category of a given state body (as a 'government department', 'state agency' or 'other').
+* ``formedByStatute`` SHOULD be used to give details of the legislation under which an enterprise was formed.
```

Comment:
I think we should remove the word 'enterprise' here and use entity as elsewhere.

I'm also not sure that this is a SHOULD unless we clarify that this only applies to entities that are explicitly formed by legislation - we don't want people hunting around for irrelevant legislation to fill in this field.

Reply:
I've edited that line now... to a MAY with 'entity' instead of 'enterprise'